interfaces/builtin/desktop_legacy.go

```diff
@@ -0,0 +1,236 @@
+dbus (receive)
+    bus=accessibility
+    peer=(label=unconfined),
+
+# Allow querying for capabilities and registering
+dbus (send)
+    bus=accessibility
+    path="/org/a11y/atspi/accessible/root"
+    interface="org.a11y.atspi.Socket"
+    member="Embed"
+    peer=(name=org.a11y.atspi.Registry, label=unconfined),
+dbus (send)
+    bus=accessibility
+    path="/org/a11y/atspi/registry"
+    interface="org.a11y.atspi.Registry"
+    member="GetRegisteredEvents"
+    peer=(name=org.a11y.atspi.Registry, label=unconfined),
+dbus (send)
+    bus=accessibility
+    path="/org/a11y/atspi/registry/deviceeventcontroller"
+    interface="org.a11y.atspi.DeviceEventController"
+    member="Get{DeviceEvent,Keystroke}Listeners"
+    peer=(name=org.a11y.atspi.Registry, label=unconfined),
+dbus (send)
+    bus=accessibility
+    path="/org/a11y/atspi/registry/deviceeventcontroller"
+    interface="org.a11y.atspi.DeviceEventController"
+    member="NotifyListenersSync"
+    peer=(name=org.a11y.atspi.Registry, label=unconfined),
+
+# org.a11y.atspi is not designed for application isolation and these rules
+# can be used to send change events for other processes.
+dbus (send)
+    bus=accessibility
+    path="/org/a11y/atspi/accessible/root"
+    interface="org.a11y.atspi.Event.Object"
+    member="ChildrenChanged"
+    peer=(name=org.freedesktop.DBus, label=unconfined),
+dbus (send)
+    bus=accessibility
+    path="/org/a11y/atspi/accessible/root"
+    interface="org.a11y.atspi.Accessible"
+    member="Get*"
+    peer=(label=unconfined),
+dbus (send)
+    bus=accessibility
+    path="/org/a11y/atspi/accessible/[0-9]*"
+    interface="org.a11y.atspi.Event.Object"
+    member="{ChildrenChanged,PropertyChange,StateChanged,TextCaretMoved}"
+    peer=(name=org.freedesktop.DBus, label=unconfined),
+dbus (send)
+    bus=accessibility
+    path="/org/a11y/atspi/accessible/[0-9]*"
+    interface="org.freedesktop.DBus.Properties"
+    member="Get{,All}"
+    peer=(label=unconfined),
+
+dbus (send)
+    bus=accessibility
+    path="/org/a11y/atspi/cache"
+    interface="org.a11y.atspi.Cache"
+    member="{Add,Remove}Accessible"
+    peer=(name=org.freedesktop.DBus, label=unconfined),
+
+
+# ibus
+# subset of ibus abstraction
+/usr/lib/@{multiarch}/gtk-2.0/[0-9]*/immodules/im-ibus.so mr,
+owner @{HOME}/.config/ibus/      r,
+owner @{HOME}/.config/ibus/bus/  r,
+owner @{HOME}/.config/ibus/bus/* r,
+
+# allow communicating with ibus-daemon (this allows sniffing key events)
+unix (connect, receive, send)
+    type=stream
+    peer=(addr="@/tmp/ibus/dbus-*"),
+
+
+# mozc
+# allow communicating with mozc server
+unix (connect, receive, send)
+     type=stream
+     peer=(addr="@tmp/.mozc.*"),
+
+
+# fcitx
+# allow communicating with fcitx dbus service
+dbus send
+    bus=fcitx
+    path=/org/freedesktop/DBus
+    interface=org.freedesktop.DBus
+    member={Hello,AddMatch,RemoveMatch,GetNameOwner,NameHasOwner,StartServiceByName}
```

Comment:
Hello? Wow I didn't know such a method exits. https://dbus.freedesktop.org/doc/dbus-specification.html#standard-interfaces is interesting read!